content/en/docs/components/pipelines/v2/concepts/component.md

```diff
@@ -0,0 +1,44 @@
++++
```

Comment:
This page seems to be overlapping a lot with https://www.kubeflow.org/docs/components/pipelines/v2/components/